src/modules/CartPage/CartItem/CartItem.module.scss

```diff
@@ -1,5 +1,130 @@
+@import '../../../styles/utils/variables/colors';
+@import '../../../styles/utils/mixins/mixin-typography';
+@import '../../../styles/utils/mixins/mixin-media';
+
 .cartItem {
-   display: flex;
-   gap: 24px;
-   padding: 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 1rem;
+  max-width: 350px;
+  margin: 0 auto;
+  border-radius: 1rem;
+  border: 1px solid $color__elements;
+
+  @include onTablet {
+    flex-direction: row;
+    max-width: 1200px;
+    padding: 1.5rem;
+    gap: 1.5rem;
+  }
+
+  @include onDesktop {
+    margin-right: 360px;
+    // I will remove this when I have grid
+  }
 }
+
+.top {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+
+  @include onTablet {
+    flex: 60%;
+    gap: 1.5rem;
+  }
+
+  @include onDesktop {
+    flex: 70%;
+  }
+}
+
+.bottom {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @include onTablet {
+    flex: 40%;
+  }
+
+  @include onDesktop {
+    flex: 30%;
+  }
+}
+
+.changeAmount {
+  display: flex;
+  gap: 13px;
+  align-items: center;
+}
+
+.content {
+  @include bodyText-typography;
+}
+
+.imgContainer {
+  box-sizing: border-box;
+  width: 80px;
+  min-width: 80px;
+  height: 80px;
+  padding: 7px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.img {
+  display: block;
+  margin: 0 auto;
+  width: 80%;
+  height: 100%;
+  object-fit: fill;
+
+  &__dark {
+    border-radius: 8px;
+  }
+}
+
+.closeButton {
+  border: none;
+  background-color: transparent;
+}
+
+.changeAmountButton {
+  width: 2rem;
+  height: 2rem;
+  border: 1px solid $color__elements;
+  border-radius: 50px;
+  background-color: transparent;
+
+  &__active {
+    border-color: $color__icons;
+  }
+
+  &__dark {
```

Comment:
I think that this is considered as modificator and since we agreed on how we write them, it should be __DARK, also to others add it as well. Think if active also can be a modificator and maybe also change it to ACTIVE